custom/icds_reports/tests/agg_tests/reports/test_bihar_api_mother_details.py

```diff
@@ -41,7 +41,12 @@ def test_file_content(self):
                 "hb": 2,
                 "add": datetime.date(2017, 6, 1),
                 "last_preg_tt": None,
-                "lmp": datetime.date(2016, 10, 2)
+                "lmp": datetime.date(2016, 10, 2),
+                "anc_1": None,
```

Comment:
could we update fixture to see the data here in the test to ensure this is working?

Reply:
updated